bundles/org.openhab.ui.homebuilder/web/src/restItems.js

```diff
@@ -4,19 +4,15 @@ import { floors, objects, OBJECTS_SUFFIX } from './definitions'
 export const GROUP_PREFIX = 'g';
 
 export function addFloors(floor, model) {
-    let items = [];
-    if (model.floorsCount > 1) {
-        items.push({
-            type: 'Group',
-            name: floor.abbr,
-            label: floor.name || floor.value,
-            category: model.itemsIcons ? floor.icon : '',
-            groupNames: ['Home'],
-            entryType: 'floor'
-        });
-    }
-
-    return items;
+    return [{
+        type: 'Group',
+        name: floor.abbr,
+        label: floor.name || floor.value,
+        category: model.itemsIcons ? floor.icon : '',
+        groupNames: ['Home'],
+        entryType: 'floor',
+        tags: floor.tags
```

Comment:
We should make sure that `floor.tags` (as well as `room.tags` and `object.tags`) are not `undefined`.
E.g. 
```
tags: floor.tags || []
```
Or create a reusable method, e.g. `getTags` that'd handle empty state:
```
function getTags(entry) {
    return entry.tags || [];
}
```

And then:
```
tags: getTags(floor)
...
tags: getTags(object)
```
and so on.

Right now the set of definitions is finite and static, but in the future we may handle it dynamically.